server/game/cards/19-JS/Brimstone.js

```diff
@@ -0,0 +1,28 @@
+const DrawCard = require('../../drawcard.js');
+
+class Brimstone extends DrawCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            match: card => card.getType() === 'character' && card.controller === this.controller && card.hasTrait('Sand Snake'),
+            effect: ability.effects.modifyStrength(1)
+        });
+        
+        this.action({
+            title: 'Return Sand Snake',
+            phase: 'challenge',
+            cost: [
+                ability.costs.kneelSelf(),
+                ability.costs.returnToHand(card => card.getType() === 'character' && card.controller === this.controller && card.hasTrait('Sand Snake'))
```

Comment:
Controller check should be unnecessary here - the cost system should be enforcing this.

Reply:
good catch, will fix this